budgetkey_data_pipelines/pipelines/procurement/tenders/exemptions/add_publisher_urls_resource.py

```diff
@@ -15,11 +17,24 @@
              45: []}
 
 def resource_filter(resource_data, parameters):
+    max_pages = os.environ.get("EXEMPTION_PUBLISHERS_MAX_PAGES", "")
```

Comment:
I am using parameters in the pipeline spec, see below

but, I also allow to override some parameters using environment variables - useful for testing / development

in this case, it can also be used to do the initial sync by setting EXEMPTION_PUBLISHER_MAX_PAGES=-1